packages/gatsby-source-contentful/src/normalize.js

```diff
@@ -240,30 +241,11 @@ function contentfulCreateNodeManifest({
   const createNodeManifestIsSupported =
     typeof unstable_createNodeManifest === `function`
 
-  const cacheExists = !!syncToken
+  const shouldCreateNodeManifest = isPreview && createNodeManifestIsSupported
 
-  const shouldCreateNodeManifest =
```

Comment:
For anyone on old versions of Gatsby but on the latest version of source-contentful (with this change), their builds could suddenly get very slow. I wonder if we should check the Gatsby version here 🤔

Reply:
Yeah, we should check and tell the user that they should upgrade Gatsby

Reply:
I updated it and pushed a new commit to do this.